packages/pagination/src/PaginationContent.js

```diff
@@ -41,41 +79,7 @@ const PaginationContent = ({
         hasMore={hasMore}
         dataLength={allPages.length}
       >
-        {allPages &&
```

Comment:
Not sure if there is an easy way to just create one memoized function that returns the items array for the proper wrapper.